### PR TITLE
Translations

### PR DESCRIPTION
**Pull request by Frizi**
This is a showcase implementation for translation system. Currently only Options page is translated. More changes would be needed across the app to properly translate everything.

Features include:
- detecting and using default browser language
- language switcher in options
- preserving manually chosen language in localStorage

For users of VSCode, I recommend installing [I18n Ally] extension and adding this config to your local project .vscode/settings.json:
```json
{
    "i18n-ally.localesPaths": [
        "src/i18n/locales"
    ],
    "i18n-ally.enabledParsers": [
        "ts"
    ],
    "i18n-ally.keystyle": "nested"
}
```

This allows for preview and jump-to-definition of translated text in-code among other things.

image

[I18n Ally]: https://marketplace.visualstudio.com/items?itemName=antfu.i18n-ally

### CODE REVIEW

**Comment by stared** [comment]
If we can do it gradually, yes!

Sooner or later we want to have translations. Plus, separating text from other pieces of code is a plus for me.

@KlemKlem What you think about this feature for translations?

**Comment by KlemKlem** [comment]
Sounds great! although I am not sure when we get to different language versions.

File: .gitignore [hunk 1]
```diff
@@ -3,6 +3,9 @@ node_modules
 /dist
 .firebase/*.cache
 
+# we are using yarn.lock
+package-lock.json
+
```
Comment (stared) [inline .gitignore:8]: This should be a separate commit. 

fix(gitignore) package-lock.json not tracked

File: src/i18n/pluralizationRules.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
+function slavicPluralization(choice: number, choicesLength: number): number {
```
Comment (stared) [inline src/i18n/pluralizationRules.ts:1]: This clearly requires a TSDoc comment.

**Review by stared — CHANGES_REQUESTED** [review]
Yes, we want to have it. A small docstring would be needed, otherwise let's start using that.

**Review by stared — APPROVED** [review]

**Pull request merged**